### PR TITLE
add new target GRAVITYF7

### PR DESCRIPTION
**Pull request by cvetaevvitaliy**
MCU: STM32F722RET6 216MHz 32bit
UART: 6
Baro: BMP280
Gyro: MPU6000
ESC telem: RX6
SBUS: RX1 - (inverter)
S.PORT: TX4
SmartAudio: TX3
CameraControl: PB1
Power in: 3-6S Lipo (12.6-25V)
BEC: 5V/3А, 9V/3A (9V or 5V) 
USER1 - PC2 pin
LED indicate: green (D4) power STM32 (3.3V), red (D3) status FC, red (VTX) BEC 9V
OSD: AT7456E

### CODE REVIEW

File: configs/default/FLMD-DRONIUSF7.config [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.5 Mar 16 2020 / 05:21:46 (d4e74e39c) MSP API: 1.42
+
+board_name DRONIUSF7
```
Comment (etracer65) [inline configs/default/FLMD-DRONIUSF7.config:3]: Why is the board name not "GRAVITYF7"?  Also the file is named in correctly.

File: configs/default/FLMD-DRONIUSF7.config [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 2 2048 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
```
Comment (etracer65) [inline configs/default/FLMD-DRONIUSF7.config:91]: There's no reason to disable the magnetometer. All this does is make it hard for someone if that want to add an external magnetometer later.

File: configs/default/FLMD-DRONIUSF7.config [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set baro_i2c_address = 119
```
Comment (etracer65) [inline configs/default/FLMD-DRONIUSF7.config:94]: You shouldn't need to specify the I2C address - the autodetect will deal with this.

File: configs/default/FLMD-DRONIUSF7.config [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 88
```
Comment (etracer65) [inline configs/default/FLMD-DRONIUSF7.config:100]: This is an extremely unusually low value. Are you sure this is correct? Does the board have a built-in current sensor? If not then this shouldn't be set.

File: configs/default/FLMD-DRONIUSF7.config (hunk 1 above)
Comment (cvetaevvitaliy) [reply configs/default/FLMD-DRONIUSF7.config:3]: nop! This latest config in repo, I del this config

File: configs/default/FLMD-DRONIUSF7.config (hunk 4 above)
Comment (cvetaevvitaliy) [reply configs/default/FLMD-DRONIUSF7.config:100]: nop - scale - 182

File: configs/default/FLMD-DRONIUSF7.config (hunk 3 above)
Comment (cvetaevvitaliy) [reply configs/default/FLMD-DRONIUSF7.config:94]: del string set  baro_i2c_address  = ?

**Pull request closed without merging**